fix(docs) Resolve 196 mdx lint warnings for just, quickly, actually, PostgreSQL (#47358)
supabase/supabase: apps/docs/content/guides/database/postgres/data-deletion.mdx

vacuum (verbose) logs;
```

> For reclaiming disk space (not only marking tuples as reusable), use `VACUUM FULL` — but be aware this rewrites the entire table and takes an `ACCESS EXCLUSIVE` lock:

Author: ```suggestion
To reclaim disk space rather than only marking tuples as reusable, use `VACUUM FULL`. Note that this rewrites the entire table and takes an `ACCESS EXCLUSIVE` lock:
```
